study.md

```diff
@@ -44,21 +46,25 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+<!-- A view is the added HTML/changes that occur in a SPA when the user interacts
+with the application. -->
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+<!-- The biggest one is that page does not need to reload everytime the user
+interacts with out app.  Additionally the sepration of concerns achived
+by having the front/back ends totally apparte makes transitioning models
+on either front/back end easier b/c it does not impact the other.->
 ```
 
 ## List some technologies used to make SPA's
 
 What are some of the tools used to create single page applications?
 
 ```md
-<!-- your answer here -->
+<!-- JSON, AJAX, HTML , CSS, JQUERY, JAVASCRIT-->
```

Comment:
These are critical but also front end frameworks like Ember, Angular, and React make it really easy for us to structure our SPAs.